package.json

```
@@ -75,6 +75,7 @@
       "circuit-diagram": "quantum/nodes/circuit-diagram/circuit-diagram.js",
       "script": "quantum/nodes/script/script.js",
       "grovers-algo":  "quantum/nodes/grovers-algo/grovers-algo.js"
+      "bloch-sphere": "quantum/nodes/bloch-sphere/bloch-sphere.js",
```

```suggestion
      "grovers-algo":  "quantum/nodes/grovers-algo/grovers-algo.js",
      "bloch-sphere": "quantum/nodes/bloch-sphere/bloch-sphere.js"
```
Missing comma here is causing the build to fail when it attempts to install dependencies.